core/src/main/java/de/bwaldvogel/mongo/MongoVersion.java

```diff
@@ -0,0 +1,13 @@
+package de.bwaldvogel.mongo;
+
+import java.util.List;
+
+public interface MongoVersion {
+
+    List<Integer> getVersionArray();
+
+    String toVersionString();
```

Comment:
What do you think about adding a default implementation for this method? I think it would make it more convenient for implementers of this interface.